src/Sulu/Bundle/TrashBundle/Resources/config/doctrine/TrashItem.orm.xml

```diff
@@ -14,6 +14,7 @@
         <field name="resourceSecurityObjectType" column="resourceSecurityObjectType" type="string" nullable="true" length="191"/>
         <field name="resourceSecurityObjectId" column="resourceSecurityObjectId" type="string" nullable="true" length="191"/>
         <field name="storeTimestamp" column="storeTimestamp" type="datetime_immutable" nullable="false"/>
+        <field name="defaultLocale" column="defaultLocale" type="string" nullable="true" length="191"/>
```

Comment:
Add this field to the `CREATE TABLE` statement in the `UPGRADE.md` or add a new `ALTER TABLE` statement